nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -304,6 +304,20 @@ GRANT ALL PRIVILEGES ON *.* TO 'mysql'@'localhost' WITH GRANT OPTION;
       <link linkend="opt-security.duosec.integrationKey">security.duosec.integrationKey</link>.
     </para>
    </listitem>
+
+   <listitem>
+    <para>
+      The configuration and state directories used by <literal>nixos-containers</literal>
+      have been moved from <literal>/etc/containers</literal> and <literal>/var/lib/containers</literal>
+      to <literal>/etc/nixos-containers</literal> and <literal>/var/lib/nixos-containers</literal>.
+
+      If you are bumping <option>system.stateVersion</option> to 20.09 you are responsible for migrating these directories yourself.
```

Comment:
`<xref linkend="opt-system.stateVersion" />` is better here since a user can directly navigate to the option's documentation in the rendered releasenotes :)